app.js

```diff
@@ -0,0 +1,151 @@
+let body = document.querySelector('body');
```

Comment:
za razliku od `var`, sa `let` se deklarise variabla samo ako ce se vrsiti njena redekleracija u suprotnom koristimo `const`
pr.
`let x = 1; x = x +2;`
'const pi = 3.14; const y = 2 * pi;`

Reply:
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

Reply:
Ovo se odnosi na ceo fajl